rust/processor/src/processors/token_v2_processor.rs

```diff
@@ -77,19 +78,19 @@ fn insert_to_db_impl(
     token_activities_v2: &[TokenActivityV2],
     current_token_v2_metadata: &[CurrentTokenV2Metadata],
 ) -> Result<(), diesel::result::Error> {
-    insert_collections_v2(conn, collections_v2)?;
-    insert_token_datas_v2(conn, token_datas_v2)?;
-    insert_token_ownerships_v2(conn, token_ownerships_v2)?;
-    insert_current_collections_v2(conn, current_collections_v2)?;
-    insert_current_token_datas_v2(conn, current_token_datas_v2)?;
-    insert_current_token_ownerships_v2(conn, current_token_ownerships_v2)?;
-    insert_token_activities_v2(conn, token_activities_v2)?;
-    insert_current_token_v2_metadatas(conn, current_token_v2_metadata)?;
+    insert_collections_v2(conn, collections_v2).await?;
```

Comment:
qq, we're sequentially inserting data into multiple tables under one transaction, is it doable that we use `tokio::spawn` and `select_all` to parallelize the db operations?

Reply:
oh i see the pipelining part.